Modules/include/VersionInfoProvider.h

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#pragma once
+
+#include "ApplicationModule.h"
+#include "ConfigReader.h"
+#include "ScalarAccessor.h"
+#include "VersionInfo.h"
+#include "boost/format.hpp"
```

Comment:
system headers should be included with pointy brackets instead of quotes:
```suggestion
#include <boost/format.hpp>
```

This will also change formatting (additional blank link before this system include)